docs/docs/user-manuals/how-to-use-frontend.md

```diff
@@ -0,0 +1,237 @@
+
+#### Navigation
+
+On the left side, we find the menu. There, we can navigate to the individual categories or to our wishlists. The link "All Products" leads to the list of all products, across categories. The same goes for the link behind the name or logo of the online store.
+
+#### Adding Something to the Shopping Cart
+
+1. To do this, we navigate to a product page by clicking on a listed product.
+2. Before clicking on the "ADD TO CART" button, we should enter how many times the product variant should be added to the shopping cart.
+3. Then, we click on "ADD TO CART".
+4. Once the product is in the shopping cart (also in the backend), we receive a green message at the bottom of the screen. We can close this by clicking on the close button of the individual message or by clicking on "CLOSE ALL". Note that clicking back into the online store somewhere in the screen also closes the message, but currently, there is a bug that prevents the next message from being displayed until we completely reload the page.
+5. We can ensure that the product has indeed been added to the shopping cart by navigating to the shopping cart. To do this, simply click on the "SHOPPING CART" button in the top right corner.
+6. Now, we should see the product listed in the shopping cart.
+
+#### Creating a Wishlist
+
+1. To create a wishlist, we navigate to the "Wishlists" menu item.
+2. In the top right corner, we find the "ADD WISHLIST" button. We click on it.
+3. A dialog opens; here, we can only enter the name of our new wishlist.
+4. We confirm the creation of the wishlist by clicking "SAVE".
+5. The wishlist should now be listed.
+6. When we click on the wishlist, we are taken to the wishlist page. As expected, it is empty because no product has been added yet.
+
+#### Adding Something to a Wishlist
+
+1. To do this, we navigate back to a product page.
+2. Below the "ADD TO CART" button, we find the "ADD TO WISHLIST" button.
+3. When we click on it, a dialog opens.
+4. In said dialog, we can check the wishlists to which the product should be added.
+5. Additionally, we could navigate to the wishlist overview page via "GO TO WISHLISTS" to create another wishlist.
+6. We confirm the assignment of the wishlists by clicking "SAVE".
+7. The product has been added to the wishlist. However, there is no green message displayed. If we wanted to make sure, we would need to navigate to the wishlists and check if the product is listed there.
+
+## Known Issues and Bugs
```

Comment:
I would remove this, since ideally there will be no known issues at the end of the project :worried: 
And the team know these issues from gropius anyway.

Reply:
I get that. However, these issues can be really annoying when using the Frontend and they sort of work against the steps as they are laid out in the guide. My intention was to sort of give the reader and user a heads up. Ideally then they would not be too surprised when encountering these issues. Long story short, I would like to keep it for now because I do not think that we will be able to fix all of these issues until the project's official end and I do not want to sweep anything under the carpet.